db/migrate/20210414203512_convert_primary_language_to_iso_code.rb

```diff
@@ -0,0 +1,124 @@
+class ConvertPrimaryLanguageToIsoCode < ActiveRecord::Migration[6.1]
+  def match_language(lang)
```

Comment:
minor styling thing, but can we move this method to the end of the file

Reply:
Done.